src/redux/books/books.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice, current } from '@reduxjs/toolkit';
+
```

Comment:
Kindly note that this milestone does not require the use of `@reduxjs/toolkit`. Kindly follow the given guidelines/ specification. Kindly ensure that `book.js` has:

- [ ]  `ADD_BOOK` and `REMOVE_BOOK` as action types.
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux): `app/reducer/ACTION_TYPE`
- [ ]  Sets an initial state with an empty array.
- [ ] Creates and Exports **Action Creators**. (`addBook` and `removeBook`)
- [ ]  Creates a reducer and exports it as default.

Reply:
Done all of these!